roles/openshift_health_checker/openshift_checks/docker_storage.py

```diff
@@ -34,7 +34,7 @@ def run(self, tmp, task_vars):
             }
 
         # attempt to get the docker info hash from the API
-        info = self.execute_module("docker_info", {}, task_vars)
+        info = self.execute_module("docker_info", {}, task_vars=task_vars)
```

Comment:
> In fact I'd like to internalize task_vars and tmp so we don't have to pass them around all the time.

:+1: 

The "difficulty" is that they are part of the `run` method. In terms of OOP it is undesirable to set instance variables at that point... 

The `run` method exists in the first place to mimic the API of the action plugin... but we may reconsider the whole thing if there's a better way.